crates/sui-config/src/builder.rs

```diff
@@ -36,7 +37,6 @@ use sui_types::crypto::{
 };
 use sui_types::multiaddr::Multiaddr;
 use sui_types::object::Object;
-
```

Comment:
nit: unnecessary to remove this

Reply:
Yeah not intentional; my formatter has been acting weird lately. Will revert.